pandas/core/frame.py

```diff
@@ -5205,8 +5202,10 @@ def swaplevel(self, i=-2, j=-1, axis=0):
 
         Parameters
         ----------
-        i, j : int, str (can be mixed)
-            Level of index to be swapped. Can pass level name as string.
+        i : int or str
+            Level of first index to be swapped. Can pass level name as string.
+        j : int or str
+            Level of second index to be swapped. Can pass level name as string.
```

Comment:
I'd merge those, I think the validation script should accept this, as proposed in the PR where you update it.

Reply:
Right now the proposed script does not validate any variables on the same line, just *args and **kwargs. I left it that way in case we decide we don't want people to just throw a bunch of variables on the same line. If you think that'd be fine, then I can rewrite the script update to allow for that, and change these here.

Reply:
I think the PR allowing having those together was merged, right? Can you have `i, j` instead please